test/connectivity/modules/NetworkController/Program.cs

```diff
@@ -112,5 +114,15 @@ static async Task RemoveAllControllingRules(IList<IController> controllerList, C
             Log.LogInformation($"Network is online");
             await reporter.ReportNetworkStatus(NetworkControllerOperation.RuleSet, NetworkStatus.Default, "All", true);
         }
+
+        static async Task<bool> CheckSetStatusResult(bool success, NetworkStatus status, INetworkController controller, CancellationToken cs)
+        {
+            NetworkStatus reportedStatus = await controller.GetStatus(cs);
+
+            string resultMessage = success ? "succeded" : "failed";
+            Log.LogInformation($"Command SetStatus {status} execution {resultMessage}, network status {reportedStatus}");
```

Comment:
Move this log message to the caller after calling CheckSetStatusResult method.

Reply:
this logs both the result and the network status